### ISSUE TITLE
named-grid-areas-alignment reads its properties in lower case alone

### ISSUE THREAD

**Issue by firefoxic** [stylelint-stylistic/stylelint-stylistic#614]
### What minimal example or steps are needed to reproduce the bug?

```css
a { GRID-TEMPLATE-AREAS: "a  a"
	"b b"; }
```

### What minimal configuration is needed to reproduce the bug?

```json
{
	"plugins": ["@stylistic/stylelint-plugin"],
	"rules": {
		"@stylistic/named-grid-areas-alignment": true
	}
}
```

### What did you expect to happen?

A warning about the rows, as for `grid-template-areas` in lower case: a property name is case-insensitive to CSS, and `aspect-ratio-notation` reads its property in whatever case the file spells it.

### What actually happened?

Nothing. The rule walks its properties by an exact string in lower case — `walkDecls("grid-template-areas")` up to #45, and `GRID_AREAS_PROPERTY` spelled `/^(?:grid-template-areas|grid-template|grid)$/u` since — so a declaration in any other case is passed over in silence.

The `pairs` oracle shows the other face of it over the `grid-shorthand` fixture #45's branch adds to its corpus, and over the longhand on the base just as well: with `property-case: "upper"` listed ahead of this rule, `--fix` uppercases the property first and the rule reads nothing behind it, so the two orders of the configuration leave two files and neither carries a warning.

```
["named-grid-areas-alignment","property-case"] "a { GRID-TEMPLATE-AREAS: \"a a\"\n\t\"b b\"; }\n" warnings: 0
["property-case","named-grid-areas-alignment"] "a { GRID-TEMPLATE-AREAS: \"a  a\"\n\t\"b b\"; }\n" warnings: 0
```

Measured over `e49e9fd`. The row is a `soft` one of #356's class, and it stands on the board until the property is read in any case.

### Do you have a proposal to fix the bug?

Give `GRID_AREAS_PROPERTY` the `i` flag, as `ASPECT_RATIO_PROPERTY` has, and pin it with a `reject` case in each case the tests already spell the property in — the suite holds `GrId-TeMpLaTe-ArEaS` as an `accept` case with an empty value, which passes today because the rule never reads it. A false negative of the base and not of #45: the branch keeps the reading it found, so that it carries its issue and nothing else.
